### PR TITLE
Remove services and packaging

### PR DESCRIPTION
**Pull request by elnaz**
Because we removed the endpoints from API.

### CODE REVIEW

File: lob/__init__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 #Resources
 from lob.resource import (Address, Area, BankAccount, Check, Country, Job,
-    Letter, Object, Packaging, Postcard, Route, Service, Setting, State,
+    Letter, Object, Packaging, Postcard, Route, Setting, State,
```
Comment (robinjoseph08) [inline lob/__init__.py]: looks like theres some stray packaging code thought that was removed a while ago. think you can remove all traces of that as well?

**Comment by elnaz** [comment]
@robinjoseph08 updated

**Comment by robinjoseph08** [comment]
LGTM :+1:

**Comment by brianseitel** [comment]
LGTM

**Pull request merged**